lib/migration.js

```diff
@@ -104,3 +104,59 @@ export const convertAllProjectsToPosts = () => {
     .then(() => Project.query().where('id', id).update({migrated: true})),
     Promise.resolve()))
 }
+
+export const convertPostsForTagToChildren = (tagId, parentPostId, trxOpt) => {
+  return Post.query(qb => {
+    qb.join('posts_tags', 'posts_tags.post_id', '=', 'posts.id')
+    qb.where('posts_tags.tag_id', '=', tagId)
+    qb.where(function () {
+      this.where('type', '!=', 'project')
+      .orWhere('type', null)
+    })
+  })
+  .fetchAll()
+  .then(({models}) => reduce(models, (promise, post) => promise
+    .then(() => post.save('parent_post_id', parentPostId, trxOpt)),
+    Promise.resolve())
+  )
+}
+
+const findOfficialProjectTags = (projects) => {
+  return reduce(
+    projects,
+    (promise, project) => promise
+      .then(() => PostTag.query(qb => {
+          qb.where('post_id', '=', project.id)
```

Comment:
you don't need to specify `=`; if you omit it, it's assumed that's what you mean